text/1444-union.md

```diff
@@ -244,10 +256,12 @@ A union may have trait implementations, using the same `impl` syntax as a
 struct.
 
 The compiler should provide a lint if a union field has a type that implements
-the `Drop` trait.  The compiler may optionally provide a pragma to disable that
-lint, for code that intentionally stores a type with Drop in a union.  The
-compiler must never implicitly generate a Drop implementation for the union
-itself, though Rust code may explicitly implement Drop for a union type.
+the `Drop` trait.  The explanation for that lint should include an explanation
+of the caveat documented in the section "Writing fields".  The compiler may
+optionally provide a pragma to disable that lint, for code that intentionally
```

Comment:
I think this should be fully specified one way or the other. And, nit, we don't usually use the terminology 'pragma'.